demo-server/src/main/java/com/illia/server/controller/DemoServerController.java

```diff
@@ -32,21 +34,20 @@ public ResponseEntity<String> uploadFile(@RequestParam(name = "fileName") String
   }
 
 
-  @GetMapping("/downloadFile")
-  public ResponseEntity<Object> downloadFile(@RequestParam(name = "fileName") String fileName) throws RequestProcessorException {
-    // reasonable to return ResponseEntity<FileDocument> instead? Parse on client
-    return ResponseEntity.ok().body(requestProcessor.proceedDownloadFile(fileName).getContent());
+  @GetMapping(value = "/downloadFile", consumes = {MediaType.APPLICATION_JSON_VALUE})
+  public ResponseEntity<FileDocument> downloadFile(@RequestParam(name = "fileName") String fileName) throws RequestProcessorException {
+    return ResponseEntity.ok().body(requestProcessor.proceedDownloadFile(fileName));
   }
 
 
-  @GetMapping("/count")
-  public ResponseEntity<Long> getAmount() {
+  @GetMapping(value = "/count", consumes = {MediaType.APPLICATION_JSON_VALUE})
+  public ResponseEntity<String> getAmount() {
```

Comment:
Why change it to String?

Reply:
good point, I can't think of possible reason :)